source/features/clean-repo-sidebar.css

```
@@ -0,0 +1,4 @@
+/* Hide "Readme" link made unnecessary by toggle-files-button #3580 */
+h3.sr-only + .mt-3 .muted-link[href='#readme'] {
```

```suggestion
.rgh-clean-repo-sidebar h3.sr-only + .mt-3 .muted-link[href='#readme'] {
```

This way it will not hid it if the feature is disabled